src/qt/ve_qitems_mqtt.cpp

```diff
@@ -283,7 +292,15 @@ void VeQItemMqttProducer::onConnected()
 			mMqttConnection->deleteLater();
 			mMqttConnection = nullptr;
 			setConnectionState(Idle);
+#ifdef MQTT_WEBSOCKETS_ENABLED
```

Comment:
is mqtt-over-websockets just for webassembly, and mqtt-without-websockets just for running on a desktop, and connecting to a networked cerbo?

Reply:
yes